scipy/stats/qmc.py

```diff
@@ -89,12 +89,12 @@
 in high dimensions. (R)QMC can bring a great improvement over
 MC when the functions on which it is used are not worst case.
 For instance (R)QMC can be especially effective on integrands
-that are well approximated by sums of functions of one or two
-or some small number of their input variables at a time [10]_, [11]_.
+that are well approximated by sums of functions of
+some small number of their input variables at a time [10]_, [11]_.
```

Comment:
It might also be worth noting that QMC more effective when the integrand is smooth.